public/index-style.css

```diff
@@ -0,0 +1,182 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
```

Comment:
Nice initial style sanitation. You can also leverage plugins like CSS Baseline to handle this.